eventManager.js

```diff
@@ -229,10 +211,37 @@ var EventManager = function() {
       console.log('UPDATE assignees', pullRequests[id]);
     }
 
-    if (_.isEqual(githubEvent, 'pull_request_review_comment') &&
-            _.isEqual(eventData.action, 'created')) {
-            //self.emit('open', eventData);
-    }
+    // CODE REVIEW
+
+    // if (_.isEqual(githubEvent, 'pull_request') &&
+    //         _.isEqual(eventData.action, 'review_requested')) {
+    //
+    //   utilities.mergeBasePRData(pullRequests, eventData);
+    //   var reviewer = utilities.parseUser(eventData.requested_reviewer);
+    //
+    //   if (!pullRequests[id].reviewers) {
+    //     pullRequests[id].reviewers = [];
+    //   }
+    //
+    //   pullRequests[id].reviewers.push(reviewer);
+    // }
+    //
+    // if (_.isEqual(githubEvent, 'pull_request') &&
+    //         _.isEqual(eventData.action, 'review_request_removed')) {
+    //
+    //   var reviewer = utilities.parseUser(eventData.requested_reviewer);
+    //
+    //   if (!pullRequests[id].reviewers) {
+    //     pullRequests[id].reviewers = [];
+    //   }
+    //
+    //   pullRequests[id].reviewers.push(reviewer);
+    // }
+    //
+    // if (_.isEqual(githubEvent, 'pull_request_review_comment') &&
+    //         _.isEqual(eventData.action, 'created')) {
+    //         //self.emit('open', eventData);
+    // }
```

Comment:
I want to keep this inside for now (even though it's ugly, I know) since I can't continue to work on this until github fixes their bug.

But hopefully this will be commented out soon